### DISCUSSION TITLE
Ability to reference script files in the script scheduler

### DISCUSSION THREAD

**Discussion by BigBoooty** [ORelio/Minecraft-Console-Client#2381] [Q&A]
Recently updated the MCC to most recent version and noticed the config file is redesigned.

Short Question: Can i reference other files in the script scheduler (using the latest version of MCC)

Before in the script scheduler, i used to be able to do script=script.txt (and reference a file in the folder to run).

Am i able to still do anything similar? I want to run .cs files from the scheduler again, especially now the new version of MCC has inventory handling.

I have tried running the command send /script script.txt but it skips the mcc commands and goes straight to the server side commands, which it obvs returns 'unknown command'.

**Comment by breadbyte** [comment]
`send /script script.txt` sends the command `/script script.txt` to the server.
You'd want to remove the send and just do `script script.txt`.
